companies-manifest.js

```diff
@@ -853,6 +854,26 @@ const companies = [
       },
     },
   },
+  {
+    key: 'https://swissdevjobs.ch',
+    similarWebGlobalRank: 583400,
+    name: 'SwissDev Jobs',
+    logo: SwissDevJobsLogo,
+    style: {
+      bottom: '0.3rem',
+      height: '2.5rem',
+    },
+    projects: {
+      swissdevjobs: {
+        title: 'SwissDev Jobs transparent tech job board',
```

Comment:
I think it would make more sense to only have the name of the website as the title and the description as the `description` field.